packages/0xcert-ethereum-bitski-frontend-provider/src/core/provider.ts

```diff
@@ -0,0 +1,159 @@
+import { GenericProvider, SignMethod } from '@0xcert/ethereum-generic-provider';
+
+/**
+ * Configuration interface for generic provider.
+ */
+export interface BitskiProviderOptions {
+
+  /**
+   * Type of signature that will be used in making claims etc.
+   */
+  signMethod?: SignMethod;
+
+  /**
+   * List of addresses where normal transfer not safeTransfer smart contract methods will be used.
+   */
+  unsafeRecipientIds?: string[];
+
+  /**
+   * Source where assetLedger compiled smart contract is located.
+   */
+  assetLedgerSource?: string;
+
+  /**
+   * Source where valueLedger compiled smart contract is located.
+   */
+  valueLedgerSource?: string;
+
+  /**
+   * Number of confirmations (blocks in blockchain after mutation is accepted) are necessary to mark a mutation complete.
```

Comment:
`* Number of confirmations (blocks in blockchain after mutation is accepted) that are necessary to mark a mutation complete.`